libs/libcommon/src/libcommon/queue.py

```diff
@@ -245,34 +248,78 @@ class JobTotalMetricDocument(Document):
     created_at = DateTimeField(default=get_datetime)
 
     meta = {
-        "collection": QUEUE_METRICS_COLLECTION,
+        "collection": TYPE_AND_STATUS_JOB_COUNTS_COLLECTION,
         "db_alias": QUEUE_MONGOENGINE_ALIAS,
         "indexes": [("job_type", "status")],
     }
     objects = QuerySetManager["JobTotalMetricDocument"]()
 
 
-def _update_metrics(job_type: str, status: str, increase_by: int) -> None:
+class WorkerSizeJobsCountDocument(Document):
+    """Metric that counts the number of (waiting) jobs per worker size.
+
+    A worker size is defined by the job difficulties it handles. We hardcode
+        - light: difficulty <= 40
+        - medium: 40 < difficulty <= 70
+        - heavy: 70 < difficulty
+
+    Args:
+        worker_size (`WorkerSize`): worker size
+        jobs_count (`int`): jobs count
+        created_at (`datetime`): when the metric has been created.
+    """
+
+    id = ObjectIdField(db_field="_id", primary_key=True, default=ObjectId)
+    worker_size = EnumField(WorkerSize, required=True, unique=True)
+    jobs_count = IntField(required=True, default=0)
+    created_at = DateTimeField(default=get_datetime)
+
+    @staticmethod
+    def get_worker_size(difficulty: int) -> str:
+        if difficulty <= 40:
+            return WorkerSize.light
+        if difficulty <= 70:
+            return WorkerSize.medium
+        return WorkerSize.heavy
```

Comment:
Fix typing by returning strings instead of type WorkerSize

```suggestion
    @staticmethod
    def get_worker_size(difficulty: int) -> str:
        if difficulty <= 40:
            return WorkerSize.light.value
        if difficulty <= 70:
            return WorkerSize.medium.value
        return WorkerSize.heavy.value
```